experiments/Train.ipynb

```diff
@@ -286,8 +284,8 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "CPU times: user 37.1 s, sys: 16.1 s, total: 53.3 s\n",
-      "Wall time: 8.21 s\n"
+      "CPU times: user 37.2 s, sys: 16.5 s, total: 53.7 s\n",
+      "Wall time: 8.34 s\n"
```

Comment:
I don't know if it's the correct place to comment but I just noticed that when I run the cell than creates the embeddings out of the `product_name` feature  I have this warning:  
```WARNING:tensorflow:Layer lstm will not use cuDNN kernels since it doesn't meet the criteria. It will use a generic GPU kernel as fallback when running on GPU.```
After some research I found out that the criteria ```recurrent_dropout``` was not compatible with the cuDNN GPU heavy parallelization so I wonder if we could just set this criteria to 0 for the sake of efficiency. It doesn't appear vital to me. 

````
product_name_graph = layers.Bidirectional(layers.LSTM(
    units = 64,
    recurrent_dropout = 0.2, # <---- HERE maybe set recurrent_dropout = 0
```